sdks/java/core/src/test/java/org/apache/beam/sdk/coders/CoderRegistryTest.java

```diff
@@ -89,8 +90,14 @@ public void testSerializableFallbackCoderProvider() throws Exception {
   public void testProtoCoderFallbackCoderProvider() throws Exception {
     CoderRegistry registry = getStandardRegistry();
 
+    //must use reflection here or a reference to dataflow class is baked into
```

Comment:
To make sure I understand correctly: are you saying that when not specifying a profile, all `CoderRegistryTest` tests run during the `runnable-on-service-tests` execution? It should execute just the two marked `Category(NeedsRunner.class)`-- this is the behavior I observe.

I would be ok with the java8 requirement for Tycho 0.25. Note that [according to the release notes](https://wiki.eclipse.org/Tycho/Release_Notes/0.25), we can still build against other JDKs.